client/app/routes/dashboard.tsx

```
@@ -0,0 +1,24 @@
+import { apiFetch } from "@/lib/utils";
+import type { Route } from "./+types/dashboard";
+
+export async function clientLoader({}: Route.ClientLoaderArgs) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)